awsx/s3/bucket.ts

```diff
@@ -16,7 +16,7 @@ import * as aws from "@pulumi/aws";
 import * as pulumi from "@pulumi/pulumi";
 import { ResourceOptions } from "@pulumi/pulumi";
 import * as schema from "../schema-types";
-import { getRegion, getRegionFromOpts, parseArn } from "../utils";
```

Comment:
Cleanup? Some missing liner rules?

Reply:
Yeah, I'm currently trying to see which one we're missing